src/form.js

```diff
@@ -185,6 +185,12 @@ define([
         removeHashtag: function(hashtag) {
             delete this.hashtagDictionary[hashtag];
         },
+        clearNullHashtags: function () {
+            this.hashtagDictionary = _.chain(this.hashtagDictionary)
```

Comment:
Hmm...I would not support switching to lodash unless either we also switched HQ or there's something desperately awesome about lodash. Would support upgrading underscore to 1.8, though.